tests/virt_autotest/virt_v2v_src.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This test verifies sle and windows guest migration from xen to kvm using virt-v2v.
+#          This is the part to run on source host.
+# Maintainer: alice <xlai@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use testapi;
+use lockapi;
+use mmapi;
+
+sub run() {
+    my $self = shift;
```

Comment:
We have it documented for -15m already!: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2233